package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "practicing DOM manipulation by dynamically rendering a simple restaurant homepage",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
Necessary run common commands like build, development should be in package.json inside `scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from Pathwright).